spec/features/graphql_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "spec_helper"
+
+RSpec.describe "Api" do
+
+  context 'when using only graphql' do
+    before :all do
+      drop_dummy_database
+      remove_project_directory
+      create_dummy_project("api" => :graphql)
+    end
+
+    it "adds graphql related gems to Gemfile" do
+      gemfile_content = IO.read("#{project_path}/Gemfile")
+      expect(gemfile_content).to include("gem 'graphql'")
+      expect(gemfile_content).to include("gem 'graphql_playground-rails'")
+      expect(gemfile_content).not_to include("gem 'graphiql'")
+    end
+
+    it 'installs and sets up graphql' do
+      schema = IO.read("#{project_path}/app/graphql/#{PotassiumTestHelpers::APP_NAME.dasherize.gsub('-','_')}_schema.rb")
```

Comment:
Use tr instead of gsub.<br>Line is too long. [121/100]<br>Space missing after comma.